spec/inverted-index-helper.js

```diff
@@ -1,25 +1,55 @@
-class invertedIndexHelper {
+/**
+ * InvertedIndexHelper
```

Comment:
So you can actually `export` the class and `import` from `jasmine`, while also attaching the class to `window`.